
global_frame: /map
robot_base_frame: /base_link
transform_tolerance: 0.2
robot_radius: 0.3

update_frequency: 5.0
publish_frequency: 2.0

rolling_window: false
static_map: true

width: 7.15
height: 13.95
resolution: 0.05
origin_x: 0
origin_y: 0


plugins:
  - {name: static_map, type: "costmap_2d::StaticLayer"}
  - {name: inflation, type: "costmap_2d::InflationLayer"}

static_map:
  unknown_cost_value: -1
  lethal_cost_threshold: 100
  map_topic: /map

inflation: 
  inflation_radius: 0.3
  cost_scaling_factor: 6
  enabled: true
  inflate_unknown: false
  
